webui/react/src/components/kit/CodeEditor.tsx

```diff
@@ -132,36 +146,32 @@ const isConfig = (key: unknown): key is Config =>
  *
  * experimentID: the experiment ID;
  *
+ * files: an array of one or more files to display code;
+ *
+ * onSelectFile: called with filename when user changes files in the tree;
+ *
  * submittedConfig: the experiments.original_config property
  *
  * runtimeConfig: the config corresponding to the merged runtime config.
  */
 
-const CodeViewer: React.FC<Props> = ({
+const CodeEditor: React.FC<Props> = ({
+  editable,
   experimentId,
```

Comment:
can we pull out the experiment-specific code? Designkit elements shouldn't rely on api or store functionality.